library/src/main/scala/scala/scalajs/js/Object.scala

```diff
@@ -57,6 +57,38 @@ object Object extends Object {
   def apply(): Object = native
   def apply(value: Any): Object = native
 
+  /** <span class="badge badge-ecma6" style="float: right;">ECMAScript 6</span>
+   *
+   * The Object.assign() method is used to copy the values of all enumerable own
+   * properties from one or more source objects to a target object. It will
+   * return the target object.
+   *
+   *
+   * Properties in the target object will be overwritten by properties in the
+   * sources if they have the same key.  Later sources' properties will similarly
+   * overwrite earlier ones.
+   *
+   * The Object.assign() method only copies enumerable and own properties from a
+   * source object to a target object. It uses [[Get]] on the source and [[Set]]
```

Comment:
Those are probably going to cause errors when generating Scaladoc. Surround them with backticks if necessary. Also in general, it's recommended to surround code with backticks so that it renders appropriately.